src/Kestrel.Core/Internal/Infrastructure/IThreadPool.cs

```diff
@@ -7,9 +7,9 @@
 
 namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure
 {
-    public interface IThreadPool : IScheduler
+    public abstract class IThreadPool: Scheduler
```

Comment:
Rename to something else?

Reply:
Yep, I'll clean up the code, just wanted to see if I broke anything large during  refactorings.